cdap-ui/app/cdap/components/FileBrowser/FilePath/FilePath.scss

```diff
@@ -36,4 +37,35 @@ $path-color: #333333;
     margin-left: 5px;
     margin-right: 5px;
   }
+
+  .collapsed-dropdown,
+  .paths {
+    display: inline-block;
+  }
+
+  .collapsed-dropdown {
```

Comment:
Might want to give a padding or have ellipsis on longer folder names.

Noticed this one
<img width="428" alt="screen shot 2017-04-23 at 10 42 55 pm" src="https://cloud.githubusercontent.com/assets/1452845/25323735/498dddaa-2876-11e7-82d6-c430c625e0e6.png">